app/Http/Controllers/Admin/AdminController.php

```diff
@@ -15,21 +39,143 @@ class AdminController extends Controller
      */
     public function index()
     {
-        return view('admin.index');
+        $roleAdmin = \App\User::ADMIN;
+        return view('admin.user.dashboard', ['roleAdmin' => $roleAdmin]);
     }
 
+    /**
+     * Log account admin out of the application.
+     *
+     * @return void
+     */
     public function userList()
     {
-        return view('admin.user.list');
+        $users = Auth::user();
+        $roleAdmin = \App\User::ADMIN;
+        if ($users->role === \App\User::ADMIN) {
+            $users = User::where('role', \App\User::ROLE_USER)
+                ->paginate(\App\User::PAGINATE_USER);
+            return view(
+                'admin.user.list', [
+                'users' => $users, 'roleAdmin' => $roleAdmin
+                ]
+            );
+        } else {
+            return redirect()->back();
+        }
     }
 
+    /**
+     * Log account admin out of the application.
+     *
+     * @return void
+     */
     public function userCreate()
     {
-        return view('admin.user.createOrUpdate');
+        $roleAdmin = \App\User::ADMIN;
+        return view('admin.user.createOrUpdate', ['roleAdmin' => $roleAdmin]);
+    }
+
+    /**
+     * Log account admin out of the application.
+     *
+     * @param Request $request Request create user
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function handleUserCreate(Request $request)
+    {
+        $this->validate(
+            $request, [
+            'username' => 'required',
+            'email' => 'required|unique:users,email',
+            'password' => 'required',
+            ]
+        );
+
+        $user = new User();
+        $user->name = $request->username;
+        $user->email = $request->email;
+        $user->password = bcrypt($request->password);
+        $user->can_see = \App\User::IS_TRUE;
```

Comment:
user table not have default value for can_see , can_delete ? why need if null case in here ?

Reply:
Fixed, for case create account user